### PR TITLE
Update unit-tests documentation. closes #1530

### LINKED ISSUE #1530: Update doc/unit-tests.md
Include instructions for running Zcash-specific tests.

### PR DESCRIPTION
**Pull request by daira**
Signed-off-by: Daira Hopwood <daira@jacaranda.org>

### CODE REVIEW

File: doc/unit-tests.md [hunk 1]
```diff
@@ -1,20 +1,16 @@
...
 
-To add more bitcoind tests, add `BOOST_AUTO_TEST_CASE` functions to the existing
-.cpp files in the test/ directory or add new .cpp files that
-implement new BOOST_AUTO_TEST_SUITE sections.
+* ``qa/zcash/full-test-suite.sh``, to run the main test suite
+* ``qa/pull-tester/rpc-tests.sh``, to run the RPC tests.
 
-To run the bitcoin-qt tests manually, launch src/qt/test/test_bitcoin-qt
+The main test suite uses two different testing frameworks. Tests using the Boost
+framework are under ``src/test/``, and tests using the Google Test/Google Mock
```
Comment (str4d) [inline doc/unit-tests.md]: and ``src/wallet/test/``

File: doc/unit-tests.md [hunk 2]
```diff
@@ -1,20 +1,16 @@
...
+framework are under ``src/gtest/``. The latter is preferred for new Zcash unit tests.
```
Comment (str4d) [inline doc/unit-tests.md]: and ``src/wallet/gtest/``. The latter are preferred

**Comment by daira** [comment]
Self-ACK. @zkbot r+

**Comment by bitcartel** [comment]
Post-ACK

**Comment by daira** [comment]
@zkbot rollup

**Pull request merged**